src/reducers/index.js

```diff
@@ -0,0 +1,90 @@
+const {
+    ADD,
+    REMOVE,
+    HALT,
+    MOVE,
+    PAID,
+    RESUME,
+    SCHEDULE,
+    SKIP,
+    STATUS,
+    TERMINATE
+} = require('../commands')
+
+function lazyJar(state = {}, action) {
+    const { event } = action
+    switch (action.type) {
+        case SCHEDULE:
+            return Object.assign({}, state, {
+                [event]: {
+                    "userIds": action.userIds,
+                    "frequency": action.frequency,
+                    "time": action.time,
+                    "halted": false
+                }
+            })
+        case TERMINATE:
+            const newState = Object.assign({}, state)
+            delete newState[event]
+            return newState
+        default:
```

Comment:
instead of default, specify each action type:
```
case ADD:
case REMOVE:
....
case: MOVE:
    const { userIds, frequency, time, halted } = state[event]
    ....
```